styleguide/_themes/derek/scss/components/_basement.scss

```diff
@@ -0,0 +1,9 @@
+.basement{
+  padding: 10px 0;
+  .basement-regionSelect{
```

Comment:
No need to nest modular classes in the main module class. These classes should be out on their own as their names are unique.